ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -25,10 +26,11 @@ import org.codice.compliance.SAMLBindings_3_4_4_1_d
 import org.codice.compliance.SAMLBindings_3_4_4_1_e
 import org.codice.compliance.SAMLBindings_3_4_4_1_f
 import org.codice.compliance.SAMLBindings_3_4_4_1_g
-import org.codice.compliance.SAMLBindings_3_4_4_a
+import org.codice.compliance.SAMLBindings_3_4_4_b
```

Comment:
Added a new Spec Reference `SAMLBindings_3_4_4_a`, so I had to move this down to `SAMLBindings_3_4_4_b`.